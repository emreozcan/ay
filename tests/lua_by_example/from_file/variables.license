https://luabyexample.techplexlabs.com/variables/
Lua by Example: Variables
by Blake Bourque

This work is licensed under a Creative Commons Attribution 3.0 Unported License.