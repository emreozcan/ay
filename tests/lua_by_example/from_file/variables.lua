first = "initial"
print(first)

b, c = 1, 2
print(b, c)

local d = true
print(d)

local value = "starts as a string"
print(value)

value = 5
print(value)

value = nil
print(value)