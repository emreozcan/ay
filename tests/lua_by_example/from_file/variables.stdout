initial
1	2
true
starts as a string
5
nil
